### PR TITLE
Blizz sorc Nihlathak script

### PR DESCRIPTION
**Pull request by D2RLegit**
Updated nihlathak with early end script if cold immune. Reduced pickit timer if stuck.

### CODE REVIEW

**Review by D2RLegit — COMMENTED** [review]
Good to go. May have to test with other chars to make sure pathing isnt effected on them.

File: src/char/sorceress/blizz_sorc.py [hunk 1]
```diff
@@ -198,24 +209,25 @@ def kill_council(self) -> bool:
...
-                    rot_deg = random.randint(-10, 10) if i % 2 == 0 else random.randint(170, 190)
-                    tele_pos_abs = unit_vector(rotate_vec(cast_pos_abs, rot_deg)) * 100
-                    pos_m = self._screen.convert_abs_to_monitor(tele_pos_abs)
-                    self.pre_move()
-                    self.move(pos_m)
+                cast_pos_abs = np.array([nihlatak_pos_abs[0] * 1.0, nihlatak_pos_abs[1] * 1.0])
+                wait(0.8)  
+                self._blizzard(cast_pos_abs, spray=0)
+                wait(0.3)  
+                Nihl_immune = self._template_finder.search(["COLD_IMMUNE","COLD_IMMUNES"], self._screen.grab(), threshold=0.8, roi=self._config.ui_roi["enemy_info"]).valid
```
Comment (aeon0) [inline src/char/sorceress/blizz_sorc.py:221]: variables lower case please `nihkl_immune`

File: src/char/sorceress/blizz_sorc.py [hunk 2]
```diff
@@ -198,24 +209,25 @@ def kill_council(self) -> bool:
         return True
 
     def kill_nihlatak(self, end_nodes: list[int]) -> bool:
-        # Find nilhlatak position
-        atk_sequences = max(2, int(self._char_config["atk_len_nihlatak"]) - 1)
+        # Find nilhlatak position
```
Comment (aeon0) [inline src/char/sorceress/blizz_sorc.py:212]: remove whitespaces at end of line

**Review by aeon0 — CHANGES_REQUESTED** [review]

**Review by D2RLegit — COMMENTED** [review]
removed white space and lower cased nihl_immune

**Review by D2RLegit — COMMENTED** [review]
increased ignore item timer to 3 seconds

File: src/char/sorceress/blizz_sorc.py [hunk 3]
```diff
@@ -199,23 +210,24 @@ def kill_council(self) -> bool:
...
-                    tele_pos_abs = unit_vector(rotate_vec(cast_pos_abs, rot_deg)) * 100
-                    pos_m = self._screen.convert_abs_to_monitor(tele_pos_abs)
-                    self.pre_move()
-                    self.move(pos_m)
+                cast_pos_abs = np.array([nihlatak_pos_abs[0] * 1.0, nihlatak_pos_abs[1] * 1.0])
+                wait(0.8)  
+                self._blizzard(cast_pos_abs, spray=0)
+                wait(0.3)  
+                nihl_immune = self._template_finder.search(["COLD_IMMUNE","COLD_IMMUNES"], self._screen.grab(), threshold=0.8, roi=self._config.ui_roi["enemy_info"]).valid
+                if Nihl_immune:
```
Comment (aeon0) [inline src/char/sorceress/blizz_sorc.py:222]: you also have to adapt this variable, otherwise it will throw an error on execution.

**Review by aeon0 — CHANGES_REQUESTED** [review]

**Review by aeon0 — APPROVED** [review]

**Review by D2RLegit — COMMENTED** [review]
reviewed

**Pull request merged**